server/scripts/get-email-list.js

```diff
@@ -0,0 +1,18 @@
+const { dbClient } = require('../db');
+require('dotenv').config({ path: '../.env' });
+(async function exec(){
+    await dbClient.connect();
+    const res = await dbClient.db.query(
+   `SELECT DISTINCT body->> 'emailAddress' AS email 
+    FROM participants p 
+    JOIN participants_status ps 
+        ON  p.id = ps.participant_id AND ps.status != 'hired'
+    WHERE 
+        CAST(p.body ->> 'interested' AS TEXT) = 'yes'  
+        OR CAST(p.body->> 'interested' AS TEXT) IS NULL
+        AND p.created_at < CURRENT_TIMESTAMP - interval '6 weeks'`)
```

Comment:
@ashtonmeuser  or @Zenen-freshworks can confirm but I think we want `body->>'userUpdatedAt'` here based on AC: 
> Send to PEOIs with an updated at date past a specific window (must be able to enter 6 weeks or a date that reflects records older than 6 weeks)

Reply:
Sounds good, I'll make the change!